### PR TITLE
fix(controller): Support emissary on Windows

### PR DESCRIPTION
**Pull request by lippertmarkus**
Signed-off-by: Markus Lippert <lippertmarkus@gmx.de>

### CODE REVIEW

File: workflow/util/path/search.go [hunk 1]
```diff
@@ -3,20 +3,18 @@ package path
...
 		return name, nil
 	}
-	envPath := os.Getenv("PATH")
-	for _, dir := range strings.Split(envPath, ":") {
-		absName := filepath.Join(dir, name)
-		if _, err := os.Stat(absName); err == nil {
-			return absName, nil
-		}
+
+	if absName, err := exec.LookPath(name); err == nil {
```
Comment (alexec) [inline workflow/util/path/search.go:15]: I didn't know about this - we can refactor to remove `path.Search` - please!

Comment (alexec) [reply workflow/util/path/search.go:15]: i.e. inline

**Review by alexec — CHANGES_REQUESTED** [review]
See comment.

File: workflow/util/path/search.go (hunk 1 above)
Comment (lippertmarkus) [reply workflow/util/path/search.go]: inlined

**Comment by lippertmarkus** [comment]
@alexec having a hard time getting `make codegen` to work on my machine. Could you do that for me please?

BTW: good work on the emissary executor, almost no changes needed for Windows

**Review by alexec — APPROVED** [review]

**Pull request merged**